Include/internal/pycore_debugger_utils.h

```diff
@@ -0,0 +1,269 @@
+#ifndef Py_INTERNAL_DEBUGGER_UTILS_H
```

Comment:
I suggest to rename the file `pycore_debug_offsets.h` instead. "debug utils" is vague.

Reply:
I was going to point out that there are other debugger-related things that would go there, but your suggestion looks very correct to me. 😄  Thanks!  I'll change it.

Reply:
done